django_project/base/test/test_models.py

```diff
@@ -0,0 +1,135 @@
+# coding=utf-8
```

Comment:
I would like to see a test constraining the site geometry based on the Location Type it is linked to.

Reply:
Could you elaborate more please?